templates/layout-grid-slider.php

```diff
@@ -2,7 +2,12 @@
 namespace Netzstrategen\Gallerya;
 
 $group_size = 6;
-$image_attr = Plugin::isLazyLoadActive() ? ['data-no-lazy' => '1'] : [];
+$slider_image_size = has_image_size('post-thumbnail') ? 'post-thumbnail' : 'large';
+// Prevent wrong images height calculation caused by lazy loading.
+$image_attr = apply_filters('gallerya_lazyload_image_attributes', [
+  'data-no-lazy' => '1',
+  'class' => "no-lazy attachment-$slider_image_size size-$slider_image_size",
```

Comment:
Still wondering what the attachment- und size- class names are about, but I assume you had a reason for not documenting their usage/effect.